requirements-dev.txt

```diff
@@ -1,7 +1,7 @@
 # Python dependencies for development
 coverage != 6.3, != 6.3.*
 datasets
-hypothesis[numpy]
```

Comment:
Why is the `numpy` extra being removed?

Reply:
I thought hypotheses will cover everything including numpy, will add this as as separate comment

Reply:
Oh, right, my mistake. That extra is already covered by`pip install hypothesis`.